plugins/cloudimport/cloud_platform.py

```diff
@@ -0,0 +1,103 @@
+from abc import ABC, abstractmethod
+import requests
+from os import path
+from app.plugins import logger
+
+VALID_IMAGE_EXTENSIONS = ['.tiff', '.tif', '.bmp', '.png', '.jpeg', '.jpg', '.gif', '.svg']
+
+class CloudPlatform(ABC):
```

Comment:
Nice, I think I answered my own question from the PR thread. xD

Reply:
It should be pretty easy to support no platforms like GoogleDrive & Dropbox.

After the PR is merged, I was thinking of writting a post on the forum to explain the new plugin a little, and ask for people to add their own platforms